test/__mocks__/serial.mock.js

```diff
@@ -15,7 +15,7 @@ MockSerial.prototype.write = function write(chunk) {
 	this.data += chunk;
 };
 
-MockSerial.prototype.drain = function drain(cb) {
+MockSerial.prototype.drain = function drain(cb = () => {}) {
```

Comment:
This changes the mock interface to better match the actual interface. I considered whether that's better or if we should fix `_serialWifiConfig` to check for that error case, I did a quick check as to expected errors for `SerialPort.drain`, custom bindings and some of the low level syscalls can fail. Removing this default would mean anything tested would be forced to at least pass that callback.